.github/integrationBenchScripts/07_execBench.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#set -eo pipefail
+
+k=$1
```

Comment:
Is it the circuit degree? might need a comment

Reply:
I agree.